ZenPacks/zenoss/ZenPackLib/lib/spec/ZenPackSpec.py

```diff
@@ -289,7 +291,7 @@ def create_ordered_component_tree(self):
             if x.is_device}
 
         order = {
-            x.meta_type: float(x.order)
+            x.meta_type: float(x.scaled_order)
```

Comment:
Wrapping this in float is no longer required. x.scaled_order is guaranteed to be a float.